src/main.js

```diff
@@ -1,33 +1,29 @@
 // Create variables targetting the relevant DOM elements here 👇
+var coverImg = document.querySelector('.cover-image');
+var coverTitle = document.querySelector('.cover-title');
+var createNew = document.querySelector('.create-new-book-button');
 var formView = document.querySelector('.form-view');
+var homeButton = document.querySelector('.home-button');
 var homeView = document.querySelector('.home-view');
+var makeNew = document.querySelector('.make-new-button');
 var randomCov = document.querySelector('.random-cover-button');
 var saveCov = document.querySelector('.save-cover-button');
-var homeButton = document.querySelector('.home-button');
+var savedCoverSection = document.querySelector('.saved-covers-section');
 var savedView = document.querySelector('.saved-view');
-var makeNew = document.querySelector('.make-new-button');
-var coverImg = document.querySelector('.cover-image');
-var coverTitle = document.querySelector('.cover-title');
 var viewSaved = document.querySelector('.view-saved-button');
-var createNew = document.querySelector('.create-new-book-button');
-var tag1 = document.querySelector('.tagline-1');
-var tag2 = document.querySelector('.tagline-2');
-var savedCoverSection = document.querySelector('.saved-covers-section');
-
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```

Comment:
Do you need this line anymore? It looks like you're just starting off with one saved cover. You could remove it if you want to start with no saved covers.